blocks/inspector-controls/text-control/index.js

```diff
@@ -23,6 +23,9 @@ function TextControl( { label, value, help, instanceId, onChange, type = 'text',
 				aria-describedby={ !! help ? id + '__help' : undefined }
 				{ ...props }
 			/>
+			{ helpText ? (
```

Comment:
I think this is already what `help` does in BaseControl: https://github.com/WordPress/gutenberg/blob/master/blocks/inspector-controls/base-control/index.js#L16

Reply:
Doh, how did I miss that, it looks like you're right. I'm going to take another stab at that.